src/components/Badge/README.md

```diff
@@ -0,0 +1,18 @@
+## Badge
+An implementation of an Badge for TPAs.
+
+The Badge TPA implementation provides a few default overridable styles. By default site theme is used.
+
+## Theme properties (all are optional)
+| propName          | propType | description                                                                                                        |
+|-------------------|----------|--------------------------------------------------------------------------------------------------------------------|
+| BadgeBgColor      | string   | The color of the badge background instead of the default color for certain type                                    |
+| BadgeTextColor    | string   | The color of the bade text instead of the default color for certain type                                           |
+| BadgeBorderColor  | string   | The color of the bade border instead of the default color for certain type. Can be set only for the **Light** badge|
```

Comment:
The border is not only for the `light` priority.
It can also be changed by the user in other priorities.